pkg/logger/logger.go

```diff
@@ -172,23 +182,25 @@ func getCallerInfo(skip int) (pkg, file string, line int) {
 	return pkg, file, line
 }
 
-func (l *Logger) updateCounter(file string, line int) (new bool) {
-	return l.LogCount.update(LogOrigin{
-		File: file,
-		Line: line,
+func (l *Logger) updateCounter(file string, line int, lvl Level, msg string) {
+	l.LogCount.update(LogOrigin{
+		File:  file,
+		Line:  line,
+		Level: lvl,
+		Msg:   msg,
 	})
 }
 
-// isAggregateSetAndIsLogNotNew checks
+// isAggregateSet checks
 // - if logs aggregation is set, so:
 //   - updates log count;
-//   - returns true if the log is not new, (avoiding writing).
-func isAggregateSetAndIsLogNotNew(skip int, l *Logger) bool {
+//   - returns true (avoiding writing).
+func isAggregateSet(skip int, l *Logger, lvl Level, msg string) bool {
```

Comment:
I think `aggregateLog` would be a better name in order to indicate that this function doesn't only check for aggregation but performs the side effect of counting the log for later.